src/main/java/com/wynntils/modules/map/overlays/ui/SeaskipperWorldMapUI.java

```diff
@@ -119,24 +138,28 @@ private void updateItems() {
 
             if (stack.isEmpty() || !stack.hasDisplayName()) continue;
 
-            if (stack.getItem() == Items.NAME_TAG) receivedItems = true;
-            else continue;
+            if (stack.getItem() == Items.NAME_TAG) {
+                receivedItems = true;
 
-            String displayname = getTextWithoutFormattingCodes(stack.getDisplayName());
+                String displayname = getTextWithoutFormattingCodes(stack.getDisplayName());
 
-            if (displayname == null) continue;
+                if (displayname == null) continue;
 
-            Matcher result = SEASKIPPER_PASS_MATCHER.matcher(displayname);
-            if (!result.find()) {
-                Reference.LOGGER.info("Seaskipper Pass with name" + stack.getDisplayName() + " didn't parse");
-                continue;
-            }
+                Matcher result = SEASKIPPER_PASS_MATCHER.matcher(displayname);
+                if (!result.find()) {
+                    Reference.LOGGER.info("Seaskipper Pass with name" + stack.getDisplayName() + " didn't parse");
+                    continue;
+                }
 
-            SeaskipperLocation location = locations.get(result.group(1));
-            int cost = Integer.parseInt(result.group(2));
-            location.setCost(cost);
-            location.setActiveType(SeaskipperLocation.Accessibility.ACCESSIBLE);
-            routes.put(location, s.slotNumber);
+                SeaskipperLocation location = locations.get(result.group(1));
+                int cost = Integer.parseInt(result.group(2));
+                location.setCost(cost);
+                location.setActiveType(SeaskipperLocation.Accessibility.ACCESSIBLE);
+                routes.put(location, s.slotNumber);
+            } else if (stack.getItem() == Items.BOAT) { //Buy boat
+                boatSlot = s.slotNumber;
```

Comment:
Keeping the codebase consistent

```suggestion
                routes.put(location, s.slotNumber);
                return;
            }
            
            if (stack.getItem() == Items.BOAT) { //Buy boat
                boatSlot = s.slotNumber;
```